wiki/osu!supporter/en.md

```diff
@@ -1,41 +1,39 @@
-![osu!supporter tag](Supporter tag.png "osu!supporter tag")
+# osu!supporter
 
-Introducing
-===========
+![osu!supporter tag](Supporter_tag.png "osu!supporter tag")
```

Comment:
Delete the image and replace with this link `![osu!supporter tag](/wiki/shared/osu!supporter.png "osu!supporter tag")`